src/modules/global-messages/converters.ts

```diff
@@ -54,40 +9,31 @@ export const globalMessageConverter = {
     snapshot: QueryDocumentSnapshot<GlobalMessageType>,
   ): GlobalMessageType | undefined {
     const data = snapshot.data();
+    const potential = {
+      id: snapshot.id,
+      active: data.active,
+      title: data.title,
+      body: data.body,
```

Comment:
The `mapToLocalizedString` is redundant with the zod parsing? Haven't looked too much into what that function did.

Reply:
We could be more strict with a function like that, e.g. saying that a body requires both bokmål and English translations, but currently, with the current zod implementation, we only require that the body has at least one translation in the format of

```json
{ lang: string, value: string }
```
or 
```json
{ language: string, value: string }
```